docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,790 @@
+    "\n",
+    "$$\n",
+    "    \\left( \\frac{\\omega^2}{{k_z}^2 {v_A}^2} - 1 \\right)\n",
+    "    \\left[\\omega^2\n",
+    "        \\left(\\omega^2 - k^2 {v_A}^2 \\right)\n",
+    "        - \\beta k^2 {v_A}^2 \\left( \\omega^2 - {k_z}^2 {v_A}^2 \\right)\n",
+    "    \\right] \\\\\n",
+    "    = \\omega^2 \\left(\\omega^2 - k^2 {v_A}^2 \\right) {k_x}^2 \\left(\n",
+    "    \\frac{{c_s}^2}{{\\omega_{ci}}^2} - \\frac{c^2}{{\\omega_{pe}}^2}\n",
+    "    \\frac{\\omega^2}{{k_z}^2 {v_A}^2} \\right)\n",
+    "$$\n",
+    "\n",
+    "where\n",
+    "\n",
+    "$$\\beta = c_{s}^2 / v_{A}^2$$\n",
+    "\n",
+    "$$k_{x} = k \\sin \\theta$$\n",
+    "\n",
+    "$$k_{z} = k \\cos \\theta$$\n",
+    "\n",
+    "$$\\mathbf{B_{o}} = B_{o} \\mathbf{\\hat{z}}$$\n",
+    "\n",
+    "$\\omega$ is the wave frequency, $k$ is the wavenumber\n",
+    ", $v_{A}$ is the Alfvén velocity, $c_{s}$ is the ion\n",
+    "sound speed, $\\omega_{ci}$ is the ion gyrofrequency, and\n",
+    "$\\omega_{pe}$ is the electron plasma frequency.\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "Note\n",
+    "\n",
+    "[Hollweg 1999][hollweg1999] asserts this expression is valid for arbitrary $c_{s} / v_{A}$ and $k_{z} / k$. Contrarily, [Bellan 2012][bellan2012] states in  Section 1.7 that due to the inconsistent retention of the $\\omega / \\omega_{ci} \\ll 1$ terms the expression can only be valid if both $c_{s} \\ll v_{A}$ and the wave propagation is nearly perpendicular to the magnetic field ($|\\theta - \\pi/2| \\ll 0.1$ radians).\n",
+    "</div>\n",
+    "\n",
+    "The [hollweg()][tfds] function numerically solves for the roots of this equation, which correspond to the Fast, Alfvén, and Acoustic wave modes.\n",
+    "\n",
+    "## Contents:\n",
+    "\n",
+    "1. [Wave propogating at nearly 90 degrees](#Wave-propogating-at-nearly-90-degrees)\n",
+    "2. [Hollweg 1999 and Bellan 2012 comparsion](#Hollweg-1999-and-Bellan-2012-comparison)\n",
+    "3. [Reproduce Figure 2 from Hollweg 1999](#Reproduce-Figure-2-from-Hollweg-1999)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import astropy.units as u\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import warnings\n",
+    "\n",
+    "from astropy.constants.si import c\n",
+    "from matplotlib.ticker import MultipleLocator\n",
+    "\n",
+    "from plasmapy.dispersion.analytical.two_fluid_ import two_fluid\n",
+    "from plasmapy.dispersion.numerical.hollweg_ import hollweg\n",
+    "from plasmapy.formulary.frequencies import gyrofrequency, plasma_frequency, wc_, wp_\n",
+    "from plasmapy.formulary.lengths import inertial_length\n",
+    "from plasmapy.formulary.speeds import Alfven_speed, cs_, ion_sound_speed, va_\n",
```

Comment:
Update these imports so they all happen from `plasmapy.formulary` and use the non-alias form.  This will help with long term maintenance since imports won't be broken every time `plasmapy.formulary` is refactored.